docs/infrastructure.md

```diff
@@ -218,7 +218,7 @@ you have two general options:
 
 === "graphDB"
 
-    1. Send HTTP requests against the HTTP API of the graphDB backend 
+    1. Send HTTP requests from the neurobagel API to the HTTP REST endpoints of thethe graphDB backend 
     e.g. using `curl`. graphDB uses the [RDF4J API](https://rdf4j.org/documentation/reference/rest-api/) specification.
```

Comment:
```suggestion
    1. Send HTTP requests from the neurobagel API to the HTTP REST endpoints of the graphDB backend 
```

Reply:
did you mean I should remove the `thethe` here (not sure why codespell let me down)? Because your diff looks a little odd to me

Reply:
Yea might be the vscode plugin.